components/scream/tests/coupled/homme_shoc_cld_spa_p3_rrtmgp/CMakeLists.txt

```diff
@@ -6,12 +6,22 @@ CreateDynamicsLib("theta-l_kokkos"  4   72   35)
 
 # Create the test
 SET (TEST_LABELS "dynamics;driver;shoc;cld;p3;rrtmgp;physics")
-set (NEED_LIBS cld_fraction shoc p3 scream_rrtmgp ${dynLibName} scream_control scream_share physics_share)
-CreateUnitTest(homme_shoc_cld_p3_rrtmgp "homme_shoc_cld_p3_rrtmgp.cpp" "${NEED_LIBS}"
+set (NEED_LIBS cld_fraction shoc spa p3 scream_rrtmgp ${dynLibName} scream_control scream_share physics_share)
+CreateUnitTest(homme_shoc_cld_spa_p3_rrtmgp "homme_shoc_cld_spa_p3_rrtmgp.cpp" "${NEED_LIBS}"
                LABELS ${TEST_LABELS} 
                MPI_RANKS 1 ${SCREAM_TEST_MAX_RANKS}
-               PROPERTIES FIXTURES_SETUP homme_shoc_cld_p3_rrtmgp_generate_output_nc_files
+               PROPERTIES FIXTURES_SETUP homme_shoc_cld_spa_p3_rrtmgp_generate_output_nc_files
 )
+# Make sure the SPA data directory is correctly defined
+if (NOT EXISTS ${SCREAM_SPA_DATA_DIR})
```

Comment:
Uhm, does this mean we cannot locally test homme+physics if there's no spa data folder? This might be a valid reason for having two folders: `homme_physics` and `homme_physics_no_spa` or something. I'd like people to be able to locally test the full PD coupling scream, even if they don't have spa data...